app/models/organization.js

```diff
@@ -134,8 +135,9 @@ export default class OrganizationModel extends AgentModel {
 
   get validationSchema() {
     return super.validationSchema.append({
-      name: Joi.string().empty('').required().messages({
-        'any.required': 'Vul de naam in',
+      name: Joi.string().empty(''),
```

Comment:
**Suggestion (blocker):** In some case an input field value is null, null will be reject because it's not a string, we have `validateStringOptional` to avoid this kind of type error : https://github.com/lblod/frontend-organization-portal/blob/development/app/validators/schema.js#L96-L98